src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersSettings.jsx

```diff
@@ -171,22 +179,47 @@ class ProjectFiltersSettings extends AsyncComponent {
     onBlur(subfilters, e);
   };
 
-  renderDisabledFeature(fields) {
-    let {project, organization} = this.props;
-    return this.state.hooksDisabled.map(hook =>
-      hook(organization, project, null, fields)
-    );
-  }
-
   renderBody() {
-    let {features, params} = this.props;
+    let {organization, features, params} = this.props;
     let {orgId, projectId} = params;
     let {project} = this.state;
 
     if (!project) return null;
     let projectEndpoint = `/projects/${orgId}/${projectId}/`;
     let filtersEndpoint = `${projectEndpoint}filters/`;
 
+    let disabledAdvFilters = p => (
```

Comment:
Can we use `props` instead of `p` (or destructure in args)?

Reply:
This one's a little annoying.

 - I can't destructure into `({features})` because `features` is declared above.
 - If I name it `props` our eslinter will kick in and ask me to add `propTypes` to the component.